web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -0,0 +1,72 @@
+<template>
```

Comment:
@dchiquit I put this here in case you find it helpful, but feel free to delete as part of #188